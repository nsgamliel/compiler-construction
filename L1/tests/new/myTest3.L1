(:go
	(:go
	0 0

	rdi <- 5
	rsi <- 7
	call allocate 2
	rdi <- rax
	call print 1

	rdi <- 5
	rsi <- rax
	call allocate 2
	rdi <- rax	
	call print 1
	
	rdi <- 5
	rsi <- 3
	call allocate 2
	rdi <- rax
	call print 1

	rdi <- 5
	rsi <- rax
	call allocate 2
	rdi <- rax
	call print 1	

	return 
	)
)
