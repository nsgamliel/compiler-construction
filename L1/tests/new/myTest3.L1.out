{s:2, 3, 3}
{s:2, 0, 0}
{s:2, 1, 1}
{s:2, 0, 0}
